packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ConfigurationFiles/index.tsx

```diff
@@ -109,7 +109,11 @@ export const ConfigurationFilesComponent = ({ theme }) => {
                       {config.description}
                     </Text>
                     <Button
-                      style={{ width: 'auto' }}
+                      style={{
+                        width: 'auto',
+                        height: 'auto',
+                        padding: '2px 0',
```

Comment:
aye, not a fan of modifying padding of elements from outside :(

waiting for staging build to check out why

Reply:
![Screenshot 2020-01-31 at 14 26 47](https://user-images.githubusercontent.com/1051509/73542783-c78bba00-4435-11ea-8905-b86c5e3b2511.png)

This, want me to make a button size="small"?

Reply:
Changed to height:4